Assets/Scripts/UI/IntroHud.cs

```diff
@@ -37,6 +39,9 @@ public HudObjectValues(RectTransform rect, Vector2 show, Vector2 hide)
         private void Start()
         {
             OnOpen();
+            playButton.onClick.AddListener(() => { if (Manager.State.InIntro) Manager.State.EnterState(GameState.ToGame); });
+            creditsButton.onClick.AddListener(() => { if (Manager.State.InIntro) Manager.State.EnterState(GameState.ToCredits); });
+            quitButton.onClick.AddListener(() => { if (Manager.State.InIntro) Application.Quit(); });
```

Comment:
This isn't exactly solving our selection issue. More preventing bad things from happening when it breaks... maybe worth putting our debug here though?